### PR TITLE
Fixes #20030 - Incorrect Content Host RAM info fix

### PR DESCRIPTION
**Pull request by snagoor**
This fixes the incorrect RAM information for the following web pages 

Hosts >> Content Hosts >> Content Host Properties 
Content >> Red Hat Subscriptions >> Associations >> Hosts

### CODE REVIEW

**Comment by snagoor** [comment]
[test]

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsInfoContro
         };
 
         $scope.convertMemToGB = function (memoryValue) {
-            return (memoryValue / 1024000).toFixed(2);
+            return (memoryValue/1000000).toFixed(2);
```
Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js]: I believe 1024 is correct. If we're going to switch to 1000 it should be across all the tooling. Perhaps others have different opinions?

Comment (snagoor) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js]: @thomasmckay thanks for reviewing it. My intention is to convert the reported memory into decimal format. see [1]

 [1] - https://www.gbmb.org/kb-to-gb

Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js]: https://github.com/candlepin/candlepin/blob/master/server/src/main/resources/rules/rules.js#L543

These facts are important to display properly to users as they are related to how subscriptions are consumed. Switching unit measurement probably not a good idea. :)

Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js]: Probably switching to "memoryValue / 1024 / 1024" would be the correct change.

**Comment by jlsherrill** [comment]
@snagoor  if you make your organization membership in katello public, the  tests will run automatically :) https://help.github.com/articles/publicizing-or-hiding-organization-membership/

**Comment by jlsherrill** [comment]
[test]

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js (hunk 1 above)
Comment (snagoor) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js]: @thomasmckay ack, modified PR per your last comment **memoryValue/1048576**

**Comment by snagoor** [comment]
[test]

**Comment by jlsherrill** [comment]
couple of linting errors in jenkins:
```

00:11:25.489 app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js
00:11:25.489   134:20  error  Infix operators must be spaced  space-infix-ops
00:11:25.489 
00:11:25.489 app/assets/javascripts/bastion_katello/subscriptions/details/subscription-content-hosts.controller.js
00:11:25.489   35:20  error  Infix operators must be spaced  space-infix-ops
00:11:25.489 
```

**Comment by swapab** [comment]
[test]

**Comment by snagoor** [comment]
[test]

**Comment by jlsherrill** [comment]
Looks like there is a typo here: https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html#L115

that prevents this from working properly.  do you mind fixing that?  Should be `convertMemToGB` not `convertMemtoGB`

**Comment by jlsherrill** [comment]
Everything else looks fine!

**Comment by snagoor** [comment]
@jlsherrill ack, fixed typo `convertMemToGB` in https://github.com/snagoor/katello/blob/57d0c18e686be73ad5cbaf2e10706a2685bb7c79/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html#L115

**Comment by jlsherrill** [comment]
[test]   this error should be fixed in master

**Pull request merged**